drivers/Makefile.dep

```diff
@@ -342,6 +342,13 @@ ifneq (,$(filter my9221,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
+ifneq (,$(filter nrf802154,$(USEMODULE)))
```

Comment:
From a structural point of view, I would move this dependency into the CPUs Makefile.dep. On paper this is of course a driver, but it is completely CPU related and none of its code resides in the `/driver` code path, so I'd say its dependency defs should also not reside here.

Reply:
I get the reasoning, but which Makefile.dep do you mean exactly?